src/CopyrighFixer/CopyrighFixer/worker.h

```diff
@@ -9,6 +9,9 @@
 #define WORKER_H
 
 #include "CopyrighFixer_global.h"
+#include "config.h"
+#include "signer.h"
+#include "configparser.h"
```

Comment:
This included is redundant. Use the [fast forward declaration](https://stackoverflow.com/questions/9906402/should-one-use-forward-declarations-instead-of-includes-wherever-possible).
```suggestion

class ConfigParser;
class Signer;
class Config;
```